hedyorg/hedy
# [UI] Add slides to the repo #3749

**Description**

Fixes Issue #3749 (https://github.com/hedyorg/hedy/issues/3749) 

* Created new HTML files that embed the slides directly on the website
* Uses Flask to render the templates and create separate routes for each slide deck
* Translates one slide's Dutch content into nl.yaml file
* Testing: all unit tests are passing after changes

**How to test**

* Go to http://localhost:8080/slides/1 and see that slides from en.yaml are being served.

**Todos still open**

* Properly cache
* Shown markdown
